.github/workflows/main.yaml

```diff
@@ -282,6 +279,9 @@ jobs:
       - name: Build Bootloader
```

Comment:
While you are here .. can we rename this to "Build ELF loader" or "Build Evergreen loader" ?

Bootloader is terribly misleading

Reply:
done, i renamed everything